hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieSparkSqlWriter.scala

```diff
@@ -123,6 +126,24 @@ object HoodieSparkSqlWriter {
             streamingWritesParamsOpt: Option[StreamingWriteParams] = Option.empty,
             hoodieWriteClient: Option[SparkRDDWriteClient[_]] = Option.empty):
   (Boolean, HOption[String], HOption[String], HOption[String], SparkRDDWriteClient[_], HoodieTableConfig) = {
+    //TODO reuse DataWritingCommand sparkPlan, reduce the number of sql list in SPARK UI SQL tag, rendering raw DAG
```

Comment:
Will it incure some overhead if we don't reuse? Why not complete the TODO in this PR itself?

Reply:
@codope Sorry to reply late, no overhead, just in the SQL TAB, it doesn't look beautiful. the TODO is hard fix now, because we reproduct logical plan in hudi command plan. I will open new pr to tracking fix it.